### DISCUSSION TITLE
Merge Queue Feedback: iOS app skips the queue

### DISCUSSION THREAD

**Discussion by julioolvr** [community/community#40373] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Pretty much that. We have our main branch configured to require going through the merge queue. Today I merged a PR of mine using the iOS app and it just went through without using the queue. I would've expected either the queue to be used, or the app to fail (if it doesn't support merge queues), but not to skip the protection rule.

**Comment by ossanna16** [comment]
Hi there @julioolvr and welcome to our community! Thank you for the feedback!
